app/core/core-ui-data/src/main/kotlin/com/hedvig/android/core/uidata/UiMoney.kt

```diff
@@ -12,7 +12,7 @@ import octopus.type.CurrencyCode
 @Serializable
 data class UiMoney(val amount: Double, val currencyCode: CurrencyCode) {
   @Transient
-  val decimalFormatter: DecimalFormat = DecimalFormat("#")
+  val decimalFormatter: DecimalFormat = DecimalFormat("")
```

Comment:
What does this change do here?
Before we basically made it so that it only shows the decimal part of the number, does this now just raw print the number as it is in the UI?

Reply:
We previously rounded all amounts to nearest integer. I think we should just display the amount exactly as we get it from back-end (which should be in the format of #.##) - but if we for some reason get something else its better to show it than to round it.